pkg/auth/cli.go

```diff
@@ -24,6 +24,7 @@ type LoginOptions struct {
 	Stdin                     io.Reader // set to os.Stdin
 	Stdout                    io.Writer // set to os.Stdout
 	AcceptUnspecifiedRegistry bool      // set to true if allows login with unspecified registry
+	Verbose                   bool
```

Comment:
I don’t have a strong opinion, just to make sure this is considered:
- Should this be managed by `GetLoginFlags` to be available by all commands?
- “Verbose” is vague; is that OK or do we want a specific `--describe-location` flag? Note that this might interact with the “all commands” concern if flags collide.

Reply:
@mtrmac Sure will add this to `GetLoginFlags` for second point I am cool with it as long as everybody agrees on this `podman login --verbose` or `podman login --describe-location`. Just my two cents i think former is more generally (`verbose`) acceptable for detailed info.

Reply:
My initial reaction was “verbose makes no promise about anything”, but on reflection that’s actually pretty accurate - the API doesn’t say what kind of description is going to be returned, so “just say something more detailed” is a good way to describe what that flag does.

Reply:
Keep it verbose.

Reply:
@rhatdan Sure. @mtrmac Added to `GetLoginFlags` we dont need https://github.com/containers/podman/pull/10068 anymore :) .